lavalink/src/client.rs

```diff
@@ -72,7 +72,7 @@ pub enum ClientErrorType {
     SendingVoiceUpdate,
 }
 
-#[derive(Debug, Default)]
```

Comment:
This change should be noted in the PR description so it can be noted in the merged commit